spec/lib/appsignal/transaction_spec.rb

```diff
@@ -506,24 +538,21 @@ def []=(key, val)
           nil
         )
       end
-
-      it "should add garbage collection time" do
-        allow_any_instance_of(Appsignal::GarbageCollectionProfiler)
-          .to receive(:internal_profiler)
-          .and_return(FakeGCProfiler.new(0.12345))
-
-        transaction.finish_event('name', nil, nil, nil)
-      end
     end
 
     describe "#record_event" do
       it "should record the event in the extension" do
+        expect {
+          transaction.record_event('name', 'title', 'body', 1000, 1)
+        }.to_not raise_error
+
         transaction.ext.should_receive(:record_event).with(
           'name',
           'title',
           'body',
           1000,
-          1
+          1,
+          0
```

Comment:
we should probably at least test in these test when the GC time is > 0, that the GC value is actually send to these methods.